### PR TITLE
adjustments for new repository location

### PR DESCRIPTION
**Pull request by GitMensch**
beside other parts: leading to the build badge work again (currently failing ;-)

### CODE REVIEW

File: GameLauncher/App/Program.cs [hunk 1]
```diff
@@ -32,7 +32,7 @@ static class Program {
             try {
                 WebClientWithTimeout client = new WebClientWithTimeout();
                 File.Delete("Languages\\Default.lng");
-                client.DownloadFile("https://raw.githubusercontent.com/metonator/GameLauncher_NFSW-translations/master/Languages/English.lng", "Languages\\Default.lng");
+                client.DownloadFile("https://raw.githubusercontent.com/SoapboxRaceWorld/GameLauncher_NFSW-translations/master/Languages/English.lng", "Languages\\Default.lng");
```
Comment (metonator) [inline GameLauncher/App/Program.cs]: This is still under my repository, will return 404 not found

Comment (GitMensch) [reply GameLauncher/App/Program.cs]: Yes, correct. Before I fix this: what do you think about auto-download all languages on first start?

**Comment by metonator** [comment]
Already did in next version, but still causes some errors, so actually
using that silly "onefile" download.

W dniu środa, 31 stycznia 2018 Simon Sobisch <notifications@github.com>
napisał(a):

File: GameLauncher/App/Program.cs (hunk 1 above)
Comment (metonator) [reply GameLauncher/App/Program.cs]: Already did in next version, but still causes some errors, so actually
using that silly "onefile" download.

**Comment by GitMensch** [comment]
I've removed the translation path, the PR should be able to be merged now.

**Comment by metonator** [comment]
Merged

**Pull request merged**